cli/standalone.rs

```diff
@@ -0,0 +1,159 @@
+use crate::colors;
+use crate::flags::Flags;
+use crate::permissions::Permissions;
+use crate::program_state::ProgramState;
+use crate::tokio_util;
+use crate::worker::MainWorker;
+use deno_core::error::type_error;
+use deno_core::error::AnyError;
+use deno_core::futures::FutureExt;
+use deno_core::ModuleLoader;
+use deno_core::ModuleSpecifier;
+use deno_core::OpState;
+use std::cell::RefCell;
+use std::convert::TryInto;
+use std::env::current_exe;
+use std::fs::File;
+use std::io::Read;
+use std::io::Seek;
+use std::io::SeekFrom;
+use std::io::Write;
+use std::pin::Pin;
+use std::rc::Rc;
+
+const MAGIC_TRAILER: &[u8; 4] = b"D3N0";
```

Comment:
I'm wondering if this value should be longer, wdyt?

Reply:
![image](https://user-images.githubusercontent.com/7829205/100550654-bbea6900-327b-11eb-8e81-789dd2b55d54.png)

I think it is fine. Regular binaries seem to all have the same 16 bytes at the end. (where the EOF-3 bytes is always 0x0). I think this is enough.

Reply:
fyi this is what I get on macOS for deno 1.5.4:
![Screen Shot 2020-11-29 at 12 14 21 PM](https://user-images.githubusercontent.com/15007517/100552513-81151080-323c-11eb-8ea4-8910f13ae2b9.png)
But then I also find this counter example from python3:
![Screen Shot 2020-11-29 at 12 28 17 PM](https://user-images.githubusercontent.com/15007517/100552820-65126e80-323e-11eb-9741-f7b421d96657.png)
It seems that trailing zeroes are there just to ensure padding to a certain byte multiplier boundary. Thus I also think it we be reasonable to make the trailer longer, probably >= 64 bits (while in the crypto world, > 2^90 is when collision becomes impractical, though I guess we don't need to go to that extreme)

Reply:
It seems this is different on macOS and Linux. I'll increase it to 8 bytes.